commons/src/store/threads.ts

```diff
@@ -31,7 +36,20 @@ function initializeThreads() {
       });
       return threadsMap[queryKey];
     },
-
+    updateThread: async (
+      threadQuery: ConversationQuery,
+      queryKey: string,
+      updatedThread: Conversation,
+    ) => {
+      await updateThread(threadQuery, updatedThread).then((thread) => {
```

Comment:
nit - This can be refactored this way to avoid the nested code block
```suggestion
      const thread = await updateThread(threadQuery, updatedThread);
```